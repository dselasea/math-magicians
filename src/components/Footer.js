import React from 'react';

const Footer = () => {
  const openInTab = (url) => {
    window.open(url, '_blank', 'nonreferrer');
  };

  return (
    <footer className="footer">
      <div className="footer-container">
        <p>
          Created by
          {' '}
          <span role="link" tabIndex="0" onClick={() => openInTab('https://github.com/dselasea')} onKeyDown={() => openInTab('https://github.com/dselasea')}>Daniel Selase Anyidoho</span>
        </p>
      </div>
    </footer>
  );
};

export default Footer;
