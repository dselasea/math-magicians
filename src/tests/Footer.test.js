import renderer from "react-test-renderer";
import Footer from "../components/Footer";

test("Test Footer Component", () => {
  const component = renderer.create(<Footer />);
  const footer = component.toJSON();
  expect(footer).toMatchSnapshot();
});
