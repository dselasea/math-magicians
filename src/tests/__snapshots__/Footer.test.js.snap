// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Footer Component 1`] = `
<footer
  className="footer"
>
  <div
    className="footer-container"
  >
    <p>
      Created by
       
      <span
        onClick={[Function]}
        onKeyDown={[Function]}
        role="link"
        tabIndex="0"
      >
        Daniel Selase Anyidoho
      </span>
    </p>
  </div>
</footer>
`;
